src/services/token.js

```diff
@@ -0,0 +1,29 @@
+import 'dotenv/config';
+import Redis from 'ioredis';
+import encryption from '../helpers/encryption';
+
+const redis = new Redis();
```

Comment:
This creates a new default connection (localhost:6379)
- you can pass in a connection url, hence a hosted database: `const redisClient = new Redis(<username:password@redis_stack:port>)`
- you can also make the connection once when the application is started